molgenis-core-ui/src/main/javascript/modules/react-components/index.js

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
an index.js is basically like an index.html. It is automatically executed. It is used to produce legacy javascript bundles for the wrappers, react components, and mixins.

So if we want the entire mixin bundle, you can do require('mixins').

@fdlk can probably explain it better